api/RTCRtpReceiver.json

```diff
@@ -361,6 +361,122 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "audio_tracks_supported": {
+          "__compat": {
+            "description": "Audio tracks supported",
+            "support": {
+              "chrome": {
+                "version_added": "59"
+              },
+              "chrome_android": {
+                "version_added": "59"
+              },
+              "edge": {
+                "version_added": "12"
+              },
+              "edge_mobile": {
+                "version_added": null
+              },
+              "firefox": {
+                "version_added": "59"
+              },
+              "firefox_android": {
+                "version_added": "59",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "media.peerconnection.rtpsourcesapi.enable",
+                    "value_to_set": "true"
+                  }
+                ]
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": "46"
+              },
+              "opera_android": {
+                "version_added": "43"
+              },
+              "safari": {
+                "version_added": null
+              },
+              "safari_ios": {
+                "version_added": null
+              },
+              "samsunginternet_android": {
+                "version_added": "7.0"
+              },
+              "webview_android": {
+                "version_added": "59"
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "video_tracks_supported": {
+          "__compat": {
+            "description": "Video tracks supported",
+            "support": {
+              "chrome": {
+                "version_added": null
```

Comment:
It took me a few days to get this confirmed by the guy who implemented this in Chrome because it's less than clear in 'paperwork'. Let me run you through it. There's an inadequately named [Intent to Implement and Ship](https://groups.google.com/a/chromium.org/forum/#!msg/blink-dev/wTJMXOwcV1A/DJNDFdQPBAAJ) that adds support for video tracks. Its name implies that it only does so for `getSynchronizationSources()`. However: "This intent **also covers** (emphasis mine) expanding both APIs (`getSynchronizationSources()` and `getContributingSources()`) to work for both audio and video receivers. The previous implementation was audio-only." The [Chrome Status entry](https://www.chromestatus.com/feature/6208698208747520) says this landed in 73. (I still don't recommend taking status entries at face value. Note that I did not in this case. I've confirmed with the developer that this number is correct.)